src/tools/wasm-split/wasm-split.cpp

```diff
@@ -203,12 +201,24 @@ void writeSymbolMap(Module& wasm, std::string filename) {
   runner.run();
 }
 
-void writePlaceholderMap(const std::map<size_t, Name> placeholderMap,
-                         std::string filename) {
+void writePlaceholderMap(
+  Module& wasm,
+  const std::unordered_map<Name, std::map<size_t, Name>> placeholderMap,
+  std::string filename) {
   Output output(filename, Flags::Text);
   auto& o = output.getStream();
-  for (auto& [index, func] : placeholderMap) {
-    o << index << ':' << func << '\n';
+  for (Index i = 0; i < wasm.tables.size(); i++) {
+    const auto& table = wasm.tables[i];
+    auto it = placeholderMap.find(table->name);
+    if (it != placeholderMap.end()) {
+      o << "table " << i << " (" << table->name << ")" << "\n";
```

Comment:
I think the maps would be more readable with the `$` on the name. Also, should we only print the name if it is explicit?
```suggestion
      o << "table " << i << " ($" << table->name << ")" << "\n";
```

Reply:
WDYT about printing only explicit names?

Reply:
So when reference types is not enabled (or some emscripten condition is met) we use an existing table, so in this case there will be a single table to print placeholders on.
https://github.com/WebAssembly/binaryen/blob/9fe0d16bbb9b85df66b44a7a85c6986eede0c4e7/src/ir/module-splitting.cpp#L164-L167

And when reference types is enabled we create table per split module, but they don't have explicit names (and even if we set `hasExplicitName` to true, it wouldn't mean much, given that they are autogenerated:
https://github.com/WebAssembly/binaryen/blob/9fe0d16bbb9b85df66b44a7a85c6986eede0c4e7/src/ir/module-splitting.cpp#L230-L233

Should we just print indices only?

Reply:
Oh, good point about the explicit names not meaning much. Yes, I guess we should print only the indices.